resources/migrations/20230112091747-mediumtext.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE workflow_run change ast ast MEDIUMTEXT;
```

Comment:
```suggestion
ALTER TABLE workflow_run change ast ast MEDIUMTEXT NOT NULL;
```

otherwise the column will allow nullable values

Reply:
using `change` rather than `modify` when we only change the datatype and not the same is also a bit verbose

Reply:
Huh, I didn't know about `modify` :+1:

Reply:
Fixed in 1432299